Contributors.md

```diff
@@ -32,7 +32,7 @@ The following people have contributed to the SCAP Security Guide project
 * Olivier Bonhomme <ptitoliv@ptitoliv.net>
 * Lance Bragstad <lbragstad@gmail.com>
 * Ted Brunell <tbrunell@redhat.com>
-* Marcus Burghardt <marcus@mbsec.com.br>
```

Comment:
@marcusburghardt This change needs to be done in https://github.com/ComplianceAsCode/content/blob/master/ssg/contributors.py#L16 other wise it will be overwritten in next update.